Eastern-Research-Group/mywaterway
# Feature/819 attains download links

## Related Issues:
* [HMW-819](https://jira.epa.gov/browse/HMW-819)

## Main Changes:
* Added links to the Expert Query API & dashboard throughout the application. The former is used for direct downloads, the latter for _Advanced Filtering_. These links are placed in the following locations:
  * Community page's **Overview** tab
    * All waterbodies in watershed: direct data download
    * Each waterbody in the list: direct data download & link to portal
  * Community page's **Identified Issues** tab (**Impaired Assessed Waters** section)
    * Download data for all waterbodies in the watershed with the selected impairments
  * Community page's **Restore** tab (**Restoration Plans** section)
    * All plans: direct data download
    * Each plan: direct data download & link to portal
  * **Plan Summary** page
    * direct data download & link to portal

## Steps To Test:
1. For each of the locations listed above, test the download links and confirm that the proper parameters are being passed. Confirm the result set is showing appropriate results.
2. If there are filter controls, adjust the controls and repeat step 1.
3. If there is an _Advanced Filtering_ link, verify it links the Expert Query ATTAINS dashboard with the proper inputs populated.
